CHANGELOG.txt

```diff
@@ -1,5 +1,117 @@
 == Changelog ==
 
+= 4.0.1 - 2020-03-18 =
+
+WooCommerce
+* Enhancement - Update Action Scheduler to 3.1.4. #25966
+* Enhancement - Bump Woocommerce Admin dependency to version 1.0.2. #25961
+* Fix - Add usage_count meta before using it in a query. #25882
+* Fix - Corrected argument type validation in plugin installer API. #25858
+* Fix - Use standard admin_url function instead of absolute path. #25884
+* Fix - Reverted the performance optimization made to variation saving. #25950
+* Fix - Send dummy params to evaluate_cost method to detect validation errors. #25946
+* Fix - Made the package shipping check more permissive. #25916
+* Fix - Fix admin notes table does not exist errors when upgrading to 4.0.x. #25891
+
+WooCommerce Admin
```

Comment:
This is missing a fix: `Fix - Made the admin note loading more resilient to prevent failures when loading notes with invalid content_data. #3926`

Reply:
thanks, added in https://github.com/woocommerce/woocommerce/pull/25968/commits/8e5cf6a67749f760faadaa8716d2aef3da267dbb